pylint/checkers/variables.py

```diff
@@ -46,6 +46,7 @@
 # by astroid. Unfortunately this also messes up our explicit checks
 # for `abc`
 METACLASS_NAME_TRANSFORMS = {"_py_abc": "abc"}
+BUILTIN_ENUMERATE = "builtins.enumerate"
```

Comment:
It's used only once so we can inline. Personal preference but even if it was used twice those kind of constant have a tendencies to stay and clutter the code when we refactor and it's easier to understand what's happening if builtins are directly strings in the code. Also it's not like ``builtins.enumerate`` is going to be renamed anytime soon.

```suggestion
```